apturicovid/api/StatsClient.swift

```diff
@@ -40,8 +40,13 @@ class StatsClient: RestClient {
                 }
             }()
             
+            
+            var statsOutdated: Bool {
+                LocalStore.shared.stats == nil || LocalStore.shared.lastStatsFetchTime.distance(to: Date()) > throttle
+            }
+            
             // Complete if recently got new stats && !forced
-            guard forceFromApi || LocalStore.shared.lastStatsFetchTime.distance(to: Date()) > throttle else {
+            guard forceFromApi || statsOutdated else {
```

Comment:
The overall structure of this method looks convoluted. Can you please help to improve the readability?

First of all, the case `forceFromApi = true` means that the `ignoreOutdated` has no effect, so to make impossible impossible, the parameter of the method should be an enum fetchType: {fromApi, localKeepingOutdated, localIgnoringOutdated}`

Second, just split the function in two subfunctions: 
```
func getStats(fetchType: FetchType) -> Observable<Stats> {
    switch fetchType {
    case .fromApi:
        return getStatsFromApi()
    case .local(let ignoreOutdated: Bool):
        return getStatsFromLocalStorage(ignoreOutdated: Bool)
    }
}
```